src/Die.js

```diff
@@ -0,0 +1,11 @@
+
+export default function Die(props) {
+    const styles = {
+        backgroundColor: props.held ? "#59E391" : "white"
+    }
+    return (
+        <div className="die-face" onClick={props.hold} style={styles}>
+            <h2 className="text-3xl font-normal">{props.value}</h2>
```

Comment:
This is cool but I think using the feature of destructuring should be much better, cause it improves the readability of the code, instead of writing props. hold , you can just write hold